### PR TITLE
fix: multiple ui fixes

### PR DESCRIPTION
**Pull request by robot9706**
- Registry editor doesn't refresh after saving (shows old information)
- Notification card type tag padding fixed (+ added state)
- Version images editor "forgets" newly added images when reordering
- Reset V2, GitLab & Google registry private fields if checkbox is turned off

### CODE REVIEW

**Review by polaroi8d — APPROVED** [review]
LGTM

File: web/crux-ui/locales/en/notifications.json [hunk 1]
```diff
@@ -14,7 +15,9 @@
     "teams": "Microsoft Teams"
   },
 
+  "status": "Status:",
```
Comment (m8vago) [inline web/crux-ui/locales/en/notifications.json:18]: common:status

**Review by polaroi8d — APPROVED** [review]
Still good for me.

**Review by m8vago — APPROVED** [review]

**Comment by robot9706** [comment]
Trying to build protos... don't merge pls :)

**Comment by polaroi8d** [comment]
> Trying to build protos... don't merge pls :)

You can change the status to draft, I think.

**Pull request merged**